notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -1288,16 +1288,13 @@
     }
```

Comment:
<p>Please remove this section altogether, or create a kernel so that it can run if you want to keep it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>